### ISSUE TITLE
React Native Error

### ISSUE THREAD

**Issue by sayem314** [aadsm/jsmediatags#127]
Followed install instruction properly and tried basic example but the error is:

![image](https://user-images.githubusercontent.com/14138401/78885719-e6827a00-7a7e-11ea-9f12-e4befc216f7b.png)

**Comment by bearpeta** [comment]
same problem here

**Comment by florenzen** [comment]
I have the same problem.

The reason is, that - for whatever reason - the `jsmediatags.js` file from the `dist` directory gets bundled instead of the `jsmediatags.js` from the `build2` directory of the package. The file from `dist` is intended for use in a browser (it says `"browser": "dist/jsmediatags.js",` in `package.json`) and for React Native, we need the `"main": "build2/jsmediatags.js",`.

I was able to workaround the problem by simply removing the `browser` line from `package.json`. I am completely new to the whole JS world, so I have no clue how to fix this properly.

**Comment by tha23rd** [comment]
@florenzen thanks for the workaround! 

Would be nice for this to be fixed

**Comment by aadsm** [comment]
@florenzen do you know if this is an issue with React Native bundling the wrong file?

**Comment by florenzen** [comment]
Yes, I assume that RN packages the wrong file. But I don't know how to change the config such that the correct one is picked up.

**Comment by avi-l** [comment]
This is still a problem... 

`error: Error: While trying to resolve module `jsmediatags` from file `app/component/ExtractMetaData.js`, the package `/node_modules/jsmediatags/package.json` was successfully found. However, this package itself specifies a `main` module field that could not be resolved (`/node_modules/jsmediatags/dist/jsmediatags.js`. Indeed, none of these files exist:

  * /node_modules/jsmediatags/dist/jsmediatags.js(.native|.android.js|.native.js|.js|.android.json|.native.json|.json|.android.ts|.native.ts|.ts|.android.tsx|.native.tsx|.tsx)
  * /node_modules/jsmediatags/dist/jsmediatags.js/index(.native|.android.js|.native.js|.js|.android.json|.native.json|.json|.android.ts|.native.ts|.ts|.android.tsx|.native.tsx|.tsx)`

There is no jsmediatags.js file in the dist/ directory.
